MentoringHomeWork/HW9/AirportSimulation/Passport.cs

```diff
@@ -0,0 +1,24 @@
+using System.Net.Sockets;
+
+namespace AirportSimulation;
+
+public class Passport
+{
+    public string PassportNumber { get; set; }
+    public DateTime PassportDate { get; set; }
+    public DateTime PassportExpirationDate { get; }
+
+    public Passport(string passportNumber, DateTime passportDate, DateTime passportExpirationDate)
+    {
+        PassportNumber = passportNumber;
+        PassportDate = passportDate;
+        PassportExpirationDate = passportExpirationDate;
+
+        if (PassportExpirationDate <= DateTime.Today)
```

Comment:
```suggestion
        if (PassportExpirationDate < DateTime.Today)
```
I believe PassportExpirationDate = DateTime.Today should work for users - the last day.
And I'm not sure about the next CW ("Error: Passport has expired!") - should the user know about it? your program is still working after this message, so that's not the error - looks like it's a warning or something like that.

Reply:
resolved